hasher-matcher-actioner/hmalib/indexers/LCCIndexer.py

```diff
@@ -0,0 +1,65 @@
+from threatexchange.signal_type.pdq_index import PDQIndex
```

Comment:
Add Facebook copyright notice. Copy from any other file.

Use isort for your imports.

```sh
$ sudo apt install isort 
```

Followed by `isort filename`.

Reply:
Typical module names in python are all lower case. Since this is already in `hmalib.indexers`, maybe lcc.py would have sufficed?